check/checkvscode.go

```diff
@@ -17,35 +20,46 @@ func (c *VSCodeChecker) Check() (bool, bool) {
 		return false, true // if no VSCode is installed, skip.
 	}
 
-	cmd := exec.Command("code", "--list-extensions")
+	ok := isInstalledIn("code", "--list-extensions")
```

Comment:
I think we should remove the dependency to the `code` command. We can look in the extension directories.

Reply:
I think it's a good idea to keep this dependency, just in case the user sets a different directory as extension directory. I think that could be more common than set a different directory inside WSL.